drivers/auxdisplay/auxdisplay_serlcd.c

```diff
@@ -413,7 +407,8 @@ static const struct auxdisplay_driver_api auxdisplay_serlcd_auxdisplay_api = {
 
 #define AUXDISPLAY_SERLCD_INST(inst)                                                               \
 	static const struct auxdisplay_serlcd_config auxdisplay_serlcd_config_##inst = {           \
-		.capabilities = {                                                                  \
+		.capabilities =                                                                    \
+			{                                                                          \
```

Comment:
the coding style is to have the brace inline, and also a patch should not contain unrelated cosmetic changes, can you revert this bit please? I know clang-format gets confused by this so it has to be handled manually, sorry

Reply:
I used the .clang-format as documented here:
https://docs.zephyrproject.org/latest/contribute/guidelines.html#clang-format

```shell
clang-format -i drivers/auxdisplay/auxdisplay_serlcd.c
```

But I actually made it to be **inline** with every other line in the array (`,` at th end of each line)

Reply:
yeah it's unfortunately common, just make it back to `.capabilities = {` please.

Reply:
actually good point, the closing bracket should also be one indentation back

Reply:
you not going to like it :)

```shell
clang-format -i drivers/auxdisplay/auxdisplay_serlcd.c
```

makes all the `\` shift even more right (due to alignment) - because one of the lines got bigger.

where I had it before (matched to **all other** auxdisplay drivers) it was a a 4 lines change, now the whole structure.

Reply:
yep :-)